.travis.yml

```diff
@@ -1,44 +1,59 @@
-sudo: false
+dist: xenial
 language: python
+
 python:
-- '2.7'
-- '3.4'
-- '3.5'
-- '3.6'
-- 'pypy-5.4'
+  - '2.7'
+  - '3.4'
+  - '3.5'
+  - '3.6'
+  # - 'pypy2.7-6.0'
```

Comment:
commented out?

Reply:
Since pypy is failing anyway, I've only kept pypy3 (previously it was only pypy).

Reply:
Fair enough